### PR TITLE
add a note about removing apache thrift support

### PR DESCRIPTION
**Pull request by blampe**
@junchaowu @breerly @abhinav

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,11 +4,13 @@ Changes by Version
 0.19.0 (unreleased)
 -------------------
 
-- Reduced Zipkin submission failures to warnings.
-- **BREAKING** - removed ``tchannel.thrift.client_for`` and
+- **BREAKING** - Removed Apache Thrift support. The ``thrift`` module is no
+  longer necessary.
```
Comment (abhinav) [inline CHANGES.rst]: What ``thrift`` module? We still have the `tchannel.thrift` module for `thrift.load`.

Comment (blampe) [inline CHANGES.rst]: "``thrift`` package"?

Comment (abhinav) [inline CHANGES.rst]: "the Apache Thrift library"
?

**Comment by abhinav** [comment]
:+1:

**Comment by junchaowu** [comment]
lgtm

**Pull request merged**